MovingBlocks/Terasology
# feature(controllers): replaced Jinput controller lib with LWJGL3 (GLFW) controller lib

### Contains

Replacing Jinput lib with GLFW lib for controllers.

### How to test

1. Start terasology with pluged controller.
2. start game
3. try buttons and sticks on controller, it must works as with jinput previously.
4. unplug and plug controller during game. 

P.S. check buttons GUIDE(idk what is it), LEFT_THUMB, RIGHT_THUMB(click on stick).
There can be invalid mapping.

### Outstanding before merging
- [ ] Test mappings for axes and buttons from GLFW to Terasology.
- [ ] Implement joystick handling. (currently only gamepads)
